frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/create-local-volume-set.tsx

```diff
@@ -64,16 +64,30 @@ export const CreateLocalVolumeSet: React.FC<CreateLocalVolumeSetProps> = ({
     <>
       <LocalVolumeSetHeader />
       <div className="ceph-ocs-install__form-wrapper">
```

Comment:
This class `ceph-ocs-install__form-wrapper` is setting a flexbox, but inside we are using a Grid.
We should use just one layout.

Reply:
Removed `display: flex;` from the class.

Reply:
Ah! I commented on previous file, my bad! 
May be we can get rid of whole `div` and put the class on grid

Reply:
removed it... thanks for pointing it out...